### DISCUSSION TITLE
How to use Lexcical in vanilla browser HTML/JS without npm/other package managers?

### DISCUSSION THREAD

**Discussion by vadimkantorov** [facebook/lexical#5367] [Q&A]
https://lexical.dev/docs/getting-started/quick-start does not show a complete example of HTML/JS

How should I plug/use lexical without having to use npm/react/packaging/etc? Is it possible to just download a tarball from https://github.com/facebook/lexical/releases and then somehow plug it as `<script type="module">` in HTML?

I don't have background in web-dev and just looking to try something simple with Lexical for my personal use (without first having to grok npm/amd/esm packaging ecosystem). I saw https://github.com/facebook/lexical/issues/1707 and https://github.com/facebook/lexical/issues/1845, but still got no understanding of how to start using Lexical in vanilla set-up without any package managers.

It appears that the releases such as https://github.com/facebook/lexical/releases/tag/v0.12.5 only contain TypeScript code which is not ready for consumption by browser without preprocessing, and there are no compiled-to-javascript releases published on GitHub :(

The following `file://my_path/index.html` does not work in Chrome:
```html
<html>
    <body>
        <script src="https://cdn.jsdelivr.net/npm/lexical@0.12.5/Lexical.min.js"></script>
        <script type="module">
            import {createEditor} from '@lexical';
        </script>
    </body>
</html>
```
Error:
```
Lexical.js:8 Uncaught ReferenceError: process is not defined
    at Lexical.js:8:17
(anonymous) @ Lexical.js:8
index.html:1 Uncaught TypeError: Failed to resolve module specifier "@lexical". Relative references must start with either "/", "./", or "../".
```
<img width="645" alt="image" src="https://github.com/facebook/lexical/assets/1041752/5071888b-5658-48d3-bd32-74495e6e3567">
<img width="495" alt="image" src="https://github.com/facebook/lexical/assets/1041752/de137c60-5d4d-4c4f-95f0-97574700a3a1">
<img width="480" alt="image" src="https://github.com/facebook/lexical/assets/1041752/5ed2197a-fb68-4b39-9512-bceb0b60166f">

Also, for some reason, an error is showing in Lexical.js versioned as 0.3.13

For now, I can't get `npm` working after `sudo apt-get install npm`, which gives me `[Npm install cannot find module 'semver']` and then issues with `npm ERR! request to https://registry.npmjs.org/lexical failed, reason: Hostname/IP does not match certificate's altnames:`. It appears that webdev packaging story is kind of wary of outsiders and it not "just works"

Thanks a lot :)

**Comment by HolyOne** [comment]
Also looking for a way to run a complete rich text example pure js
